### DISCUSSION TITLE
github action runners to execution powershell script to remote machines

### DISCUSSION THREAD

**Discussion by tdubb123** [community/community#48831] [Actions]
### Select Topic Area

Product Feedback

### Body

I am testing github action runner and i want to know how i can possibly use a github runner to execute powershell commands on remote machines. The remote machines are on prem inside my network.

**Comment by jsoref** [comment]
you'd either need to use a self hosted runner or set up some sort of network bridge... It'd be the same as if you had an Azure VM and wanted to use it to trigger a powershell command on a computer in your on prem network...
